### ISSUE TITLE
The `.cmx` files should be installed as well

### ISSUE THREAD

**Issue by gasche** [ocaml/num#1]
Compiling a package that depends on `num` under recent OCaml versions currently generates loads of the following warnings:

```
File "_none_", line 1:
Warning 58: no cmx file was found in path for module Big_int, and its interface was not compiled with -opaque
File "_none_", line 1:
Warning 58: no cmx file was found in path for module Num, and its interface was not compiled with -opaque
```

**Comment by xavierleroy** [comment]
That's a serious omission indeed.  I'll look into it tomorrow.

**Comment by xavierleroy** [comment]
Fixed in release 1.1.

**Comment by xavierleroy** [comment]
OPAM packaging: https://github.com/ocaml/opam-repository/pull/10460
